parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -241,7 +241,11 @@ write_files:
     KUBELET_CONFIG={{GetKubeletConfigKeyVals .KubernetesConfig }}
     KUBELET_IMAGE={{WrapAsParameter "kubernetesHyperkubeSpec"}}
     KUBELET_REGISTER_SCHEDULABLE=true
+{{if IsKubernetesVersionGe "1.16.0-alpha.1"}}
```

Comment:
This appears to return `true` in all cases on Windows, but work as expected on Linux. I'm not sure why, but I can reproduce it with an `aks-engine generate` command and I'll try to find a fix.

Reply:
Found, fixed.